OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -66,12 +74,12 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
 		public readonly bool CanHover = false;
 
+		[Desc("List of possible action to take when becoming idle in descending order of priority. Invalid entries (e.g. Hover when CanHover is false) will be skipped.")]
+		public readonly IdleAircraftAction[] ActionsWhenIdle = { IdleAircraftAction.Land, IdleAircraftAction.Hover, IdleAircraftAction.Circle };
```

Comment:
I could see use cases where it would be nice to have an ActorInit for this, for example for aircraft spawned by scripts.